INSTALL.md

```diff
@@ -26,33 +25,41 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are four scripts in the the `bin/` directory that are use to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
+3. `accumulo-single`- Manages Accumulo on a single node
+4. `accumulo-cluster` - Manages Accumulo on a cluster
```

Comment:
Err, sorry, I wasn't clear. I meant to say that as a statement: a cluster of size 1 is still a cluster. Why do we need both accumulo-single and accumulo-cluster? Is the additional command necessary or with merit?